docs/contents/components/data-display/calendar/accessibility.ja.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Calendar
+description: "`Calendar`は、日付を表示または選択するカレンダーのコンポーネントです。"
+package_name: "@yamada-ui/calendar"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+## キーボード操作
```

Comment:
There is no explanation for `ArrowLeft` and `ArrowRight`.

Reply:
@hirotomoyamada 
I could not control the keys well.
Focus does not move in every date, move one character.

Reply:
It behaves differently depending on the software.

Read and explain the logic in the actual `onKeydown` function.